test/e2e_node/node_shutdown_linux_test.go

```diff
@@ -134,15 +134,15 @@ var _ = SIGDescribe("GracefulNodeShutdown [Serial] [NodeFeature:GracefulNodeShut
 			})
 
 			framework.ExpectNoError(err)
-			framework.ExpectEqual(len(list.Items), len(pods), "the number of pods is not as expected")
+			gomega.Expect(list.Items).To(gomega.HaveLen(len(pods)))
```

Comment:
there is another one: https://github.com/kubernetes/kubernetes/pull/121129/ doing the same. We need to keep the error descriptions. So I prefer other PR.

Thank you for the contribution!

/close